lib/models/product.rb

```diff
@@ -0,0 +1,26 @@
+module Shop
+  class Product
+    @@id = 0
+    attr_reader :name, :price, :id
+
+    def initialize(name: , price: )
+      @name = name
+      @price = set_price(price)
+      @id = set_id
+    end
+
+    def price_brutto
+      p price * 1.23
```

Comment:
Why you have `p` here? What will happen if you use `Shop::Product#price_brutto` in some calculations? will it work as expected?